packages/documentation/docs/user-guide/installation/installing-sofie-server-core.md

```diff
@@ -22,7 +22,7 @@ version: '3.3'
 services:
   db:
     hostname: mongo
-    image: mongo:4.2.18
+    image: mongo:4.4
```

Comment:
I would suggest making this 6.0, as that matches the version of mongo used by meteor in development https://v2-docs.meteor.com/changelog#v211020230302

Reply:
What I'm worried about is that the rest of this may not work in later versions? I see that mongo has removed mongo shell, whcih I think means L30 would have to be changed to `mongosh`? But that then means we require `mongosh` to be available? I just don't have the resources to verify that this will work, so I've updated the version I know works in production. If you can verify that this file indeed works with 6.0 then I'm happy to make this 6.0.

Reply:
I've given this a quick test, and the line
```
test: test $$(mongosh --quiet --eval "try {rs.initiate()} catch(e) {rs.status().ok}") -eq 1
```
looks to setup the replicaset and continue running happily with mongo:6.0